tests/Application/config/packages/doctrine.yaml

```diff
@@ -7,9 +7,9 @@ parameters:
 
 doctrine:
     dbal:
-        driver: 'pdo_sqlite'
```

Comment:
Because official skeleton brings Docker image without SQLite, but with MySQL, I have switched to it.